deployment/dataproc/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ubuntu:20.04
+
+RUN mkdir -p /usr/local/src
+WORKDIR /usr/local/src
+
+RUN apt update && apt install -y curl osmctools
```

Comment:
Most of this effort is to minimize the overhead necessary to have `osmconvert` accessible to complete a release. That binary is included in the `osmctools` package.